### PR TITLE
tstest/natlab: get tailscaled logs from gokrazy via syslog

### PR DESCRIPTION
**Pull request by bradfitz**
Using https://github.com/gokrazy/gokrazy/pull/275

This is much lower latency than logcatcher, which is higher latency
and chunkier. And this is better than getting it via 'tailscale debug
daemon-logs', which misses early interesting logs.

Updates #13038

### CODE REVIEW

File: tstest/natlab/vnet/vnet.go [hunk 1]
```diff
@@ -1248,6 +1264,14 @@ func isDNSRequest(pkt gopacket.Packet) bool {
 	return ok && dns.QR == false && len(dns.Questions) > 0
 }
 
+func isSyslog(pkt gopacket.Packet) bool {
```
Comment (maisem) [inline tstest/natlab/vnet/vnet.go:1267]: unused?

**Review by maisem — APPROVED** [review]
Why is this two commits? bad rebase?

File: tstest/natlab/vnet/vnet.go (hunk 1 above)
Comment (bradfitz) [reply tstest/natlab/vnet/vnet.go]: deleted

**Comment by bradfitz** [comment]
> Why is this two commits? bad rebase?

Not so much "bad" rebase... I just hadn't rebased since merging the other one. Done.

**Pull request merged**